free-3.md

```diff
@@ -57,12 +142,35 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think this is a 50-50 language/system project. Rendering images is the most 
+"systems" aspect. And if I make a user interface, I'll spend a lot of time 
+working on that (I don't have much front-end experience). But I will have 
+to think a lot about the underlying language design. I have to consider what is 
+"intuitive" to humans (unions vs. OR, intersections vs. AND) and how to make it 
+easy to create intersections.
+
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+Venn diagrams should be simple enough to do in a half-semester. I'm worried 
+that this project is on the boring side, but maybe that's because I'm not a 
```

Comment:
I tend to agree that this seems, not on the boring side, but on the small side. I think you could try to expand this to making general graphs and figures easily from the data, and making sure they look pleasant no matter what.